lib/linguist/heuristics.rb

```diff
@@ -68,6 +68,20 @@ def call(data)
     # Common heuristics
     ObjectiveCRegex = /^\s*(@(interface|class|protocol|property|end|synchronised|selector|implementation)\b|#import\s+.+\.h[">])/
 
+
+    CPPRegexType0 = /^\s*#.*include.*<(stdint|cstdint|string|vector|map|list|array|bitset|queue|stack|forward_list|unordered_map|unordered_set|(i|o|io)stream).*>/
```

Comment:
👋  Hey @rafael-santiago, thanks for the contribution.

Using a multiline/expanded regex is preferable to having multiple, smaller expressions. Could you combine them, please?

Additionally, this should go inside the relevant heuristic, as they're only used once. We have the `ObjectiveCRegex` assigned to a variable because it's shared by two different tests (`.h, .m`).